core/src/main/java/com/xmartlabs/base/core/Injector.java

```diff
@@ -1,11 +1,13 @@
 package com.xmartlabs.base.core;
 
 import android.support.annotation.NonNull;
+import android.support.annotation.VisibleForTesting;
 
 import bullet.ObjectGraph;
 
 public class Injector {
```

Comment:
We used `getComponent` before in the Application class, and although this case is not entirely the same, I think is better.
What do you think?